docs/sources/clients/promtail/stages/pack.md

```diff
@@ -57,7 +57,7 @@ This would create a log line
 }
 ```
 
-**Loki 2.2 also includes a new [`unpack`](../../../../logql/log_queries/#unpack) parser to work with the pack stage.**
+**Loki 2.2 also includes a new [`unpack` parser]({{< relref "../../../logql/log_queries.md#unpack" >}}) to work with the pack stage.**
```

Comment:
why `relref` here?

Reply:
`relref` is the preferred way to link documents.
First, it throws an error if the linked page does not exist, preventing 404s. And second, it is agostic about how the actual URL of the linked document looks.